CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 # bedrock-mongodb ChangeLog
 
+## 10.0.2 -
+
+### Fixed
+- No longer pass in `authSource: undefined` as this causes connection strings with auth to fail.
+- No longer pass in `ssl: undefined` in connectOptions.
+- Use `db.databaseName` when logging connection success over `config.name`.
+
+## 10.0.1 -
+
+### Fixed
+- `openDatabase` now queries for serverInfo first.
+- `openDatabase` now correctly checks if auth is needed.
+- `openDatabase` no longer double logins in authenticated users.
+
+### Added
+- `openDatabase` pings server on successful connect.
+- Github Actions now test on authenticated & unauthenticated databases.
+
```

Comment:
This needs alphabetizing and merging. Are all these things still true now that the two branches have merged?